UnityProject/Assets/Scripts/Systems/Antagonists/Objectives/EscapeObjectives/PreventEveryoneFromEscaping.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+
+
+namespace Antagonists
+{
+	[CreateAssetMenu(menuName="ScriptableObjects/AntagObjectives/PreventEveryoneFromEscaping")]
+	public class PreventEveryoneFromEscaping : Objective
```

Comment:
You need to add the InternalIsPossible override to this class and check for non Ai's so they wont get this objective

Reply:
I made it so both the AI and normal traitors can get this objective. The asset files have two versions of this. One is marked with "AI can have" and with text that states "Organic lifeforms" and the other anyone can get.

Reply:
ah ok.

Not sure whether the Ai should always gets the objective to stop all players from leaving